File: csharp/pangram/Pangram.cs
using System;
using System.Linq;

public static class Pangram
{
    public static bool IsPangram(string input) =>
        input.ToLowerInvariant()
            .Where(c => c >= 'a' && c <= 'z')
            .Distinct()
            .Count() == 26;
}

File: fsharp/pangram/Pangram.fs
module Pangram

let normalize (s: string) =
    s.ToLowerInvariant()

let contains (s: string) (c: char)  =
    let n = normalize s
    n.Contains(c)

let isPangram (input: string): bool =
    Seq.forall (input |> contains) <| ['a'..'z']